src/Pirouette/SMT/Monadic.hs

```diff
@@ -117,10 +117,10 @@ declareDatatype typeName (Datatype _ typeVariables _ cstrs) = do
       constr'
   return $ typeName : map fst cstrs
 
--- | Returns @Right ()@ when this datatype can be declared as a SMTLIB datatype
-supportedDatatype :: (LanguageSMT lang) => TypeDef lang -> Except String ()
-supportedDatatype (Datatype _ typeVariables _ cstrs) =
-  void $ runWriterT $ mapM (constructorFromPIR typeVariables) cstrs
+-- -- | Returns @Right ()@ when this datatype can be declared as a SMTLIB datatype
+-- supportedDatatype :: (LanguageSMT lang) => TypeDef lang -> Except String ()
+-- supportedDatatype (Datatype _ typeVariables _ cstrs) =
+--   void $ runWriterT $ mapM (constructorFromPIR typeVariables) cstrs
```

Comment:
Plese don't leave dead code in. You can either export this function to mean that it's part of the interface, or delete it altogether (Git can always used to retrieve the function in the future)
```suggestion
```

Reply:
Ok ! I saw some dead code [elsewhere](https://github.com/tweag/pirouette/blob/main/src/Pirouette/Transformations/Utils.hs#L25-L70) so I figured it was accepted within Pirouette.